project_setup/README.md

```diff
@@ -4,6 +4,32 @@ While the specific setup for each 18F project varies widely, there are certain
 elements that should be present on every project. This document aims to detail
 those elements and provide suggested tools and resources.
 
+### Initialization Checklist
+
+This list is aspirational, but a good place to start. We will try to include
+all in upcoming project templates; until then, do your best!
+
+1. Common project files (consider using [`18f init`](https://github.com/18F/18f-cli#18f-init)):
+    1. LICENSE.md
+    1. CONTRIBUTING.md
+    1. README.md
+    1. .about.yml
+1. Project files (e.g. package.json, setup.py) if relevant
```

Comment:
It's a good question. There are some cases (like APIs) where we wouldn't but we can highlight `package.json` as a likely necessity. We should probably also remove all of the "if applicable"/"if relevant"s and say that in the intro paragraph. I'll take a stab.